server/queries/readSurvey.test.js

```diff
@@ -1,66 +1,69 @@
 /* eslint-disable no-undef */
-// const { ObjectID } = require('mongodb');
-// const { initDb, closeDb } = require('../databaseConnection');
+const { ObjectID } = require('mongodb');
+const { initDb, closeDb } = require('../databaseConnection');
 
-// const readSurvey = require('./readSurvey');
+const readSurvey = require('./readSurvey');
 
-// beforeEach(() => {
-//   return initDb();
-// });
+beforeEach(() => {
+  return initDb();
+});
 
-// afterEach(() => {
-//   return closeDb();
-// });
+afterEach(() => {
+  return closeDb();
+});
 
 it('Testing readSurvey retrieves the correct document.', async (done) => {
   done();
```

Comment:
After done() I'm thinking the test just cuts out and doesn't even test anything?

Reply:
not my test again